com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/CopybookServiceImpl.java

```diff
@@ -80,21 +75,6 @@ public void setWorkspaceFolders(List<WorkspaceFolder> workspaceFolders) {
     this.workspaceFolders = workspaceFolders;
```

Comment:
Due to we call this setter only on the initialization, we may replace it with an assisted injection to disallow the changing of the object's internal state.

Reply:
Not sure I can apply this due to my reply [here](https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/254#discussion_r401414256)